docs/getting_started.md

```diff
@@ -3,6 +3,8 @@
 Welcome to the webKnossos documentation.
 Feel free to [drop us a line](mailto:hello@scalableminds.com) or [create a Pull Request](https://github.com/scalableminds/webknossos/pulls) if you have any suggestions for improving the documentation.
 
+{% embed url="https://www.youtube.com/watch?v=tNwDvo5MnDc "%}
+
 ## Try webKnossos
 We host a public instance of webKnossos for trial purposes at https://try.webknossos.org.
```

Comment:
Not your fault, but I just noticed that the link to the trial instance is wrong. Could you correct it to just https://webknossos.org (without the try.)? :)